website/showcase.json

```diff
@@ -319,7 +319,7 @@
       "icon": "jdcom.png",
       "linkAppStore": "https://itunes.apple.com/cn/app/shou-ji-jing-dong-xin-ren/id414245413?mt=8",
       "linkPlayStore": "https://app.jd.com/android.html",
-      "infoLink": "http://ir.jd.com/phoenix.zhtml?c=253315&p=irol-homeProfile",
+      "infoLink": "https://ir.jd.com/phoenix.zhtml?c=253315&p=irol-homeProfile",
```

Comment:
This is also broken (404).
You may ping someone from `jd(dot)com` to update this.